emc-metalnx-services/src/main/java/com/emc/metalnx/services/irods/CollectionServiceImpl.java

```diff
@@ -390,9 +390,14 @@ public List<DataGridCollectionAndDataObject> getSubCollectionsUnderPath(String p
 
 		return dataGridCollectionAndDataObjects;
 	}
-
+	
 	@Override
 	public boolean createCollection(DataGridCollectionAndDataObject collection) throws DataGridException {
+		return createCollection(collection, false);
+	}
+
+	@Override
+	public boolean createCollection(DataGridCollectionAndDataObject collection, boolean recursiveOpr) throws DataGridException {
```

Comment:
Consider renaming `recursiveOpr` to something that expresses what it does. For example, `createParentCollections`.